manifests/k/Kubernetes/kubectl/1.24.3/Kubernetes.kubectl.locale.en-US.yaml

```diff
@@ -0,0 +1,31 @@
+# Created with YamlCreate.ps1 v2.1.3 $debug=NVS1.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Kubernetes.kubectl
+PackageVersion: 1.24.3
+PackageLocale: en-US
+Publisher: Kubernetes
+PublisherUrl: https://kubernetes.io
+PublisherSupportUrl: https://github.com/kubernetes/kubectl/issues
+# PrivacyUrl: 
+Author: The Kubernetes Authors
+PackageName: kubectl
+PackageUrl: https://kubernetes.io/docs/reference/kubectl/
+License: Apache License 2.0
+LicenseUrl: https://github.com/kubernetes/kubectl/blob/master/LICENSE
+Copyright: Copyright (c) The Kubernetes Authors
+# CopyrightUrl: 
+ShortDescription: The Kubernetes command-line tool
+# Description: 
+Moniker: kubectl
+Tags:
+- kubectl
+- kubernetes
```

Comment:
```suggestion
# Tags:
```
Moniker and Package Identifier will take precedence over the tags, so these tags are redundant